src/components/FormResult.tsx

```diff
@@ -1,9 +1,14 @@
-export const FormResult = () => {
+type resultsProps = {
+  ageResults: { years: number; months: number; days: number } | null;
+}
+
+export const FormResult = ({ageResults}:resultsProps) => {
```

Comment:
Solo procura poner espacios cuando vayas a destructurar tus props. A veces hay devs algo pickys en ese tema. 
```suggestion
export const FormResult = ({ ageResults }: resultsProps) => {
```